resources/extruders/voron2_extruder_0.def.json

```diff
@@ -0,0 +1,17 @@
+{
+    "id": "voron2_extruder_0",
```

Comment:
The tests didn't pass.

```
>       assert "id" not in doc, "Definitions should not have an ID field."
E       AssertionError: Definitions should not have an ID field.
E       assert 'id' not in {'id': 'voron2_extruder_0', 'inherits': 'fdmextruder', 'metadata': {'machine': 'voron2_base', 'position': '0'}, 'name': 'Toolhead', ...}
```

Reply:
Nice, missed this. I guess this is new from 4.4. Fixed.